client/src/js/samples/components/General.js

```diff
@@ -3,8 +3,9 @@ import React from "react";
 import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 import styled from "styled-components";
-import { BoxGroup, BoxGroupHeader, Table } from "../../base";
+import { NotesBox, BoxGroup, BoxGroupHeader, Table } from "../../base";
 import { getLibraryTypeDisplayName } from "../utils";
+import { ReleaseMarkdown } from "../../updates/components/Markdown";
```

Comment:
`ReleaseMarkdown` should be turned into a base component as it is used in two unrelated parts of the application. It may have to refactored to be made more reuseable.